docs/_includes/embed.njk

```diff
@@ -20,7 +20,7 @@ title: Component Example
 <!-- Example End -->
 <script src="{{'/jazz_behavior.js' | url }}"></script>
 <script>
-    PrincetonDesignSystem.enable();
+    PrincetonDesignSystem.enable({disablePager: true});
```

Comment:
Why does the pager now need to be disabled?

Reply:
This is for the documentation site. Currently the pager component builds the content of the pager (the list items - li) based on the total pages and current page values. I wanted to show in the documentation site the html needed to populate the pager so for this i disabled the pager component. I think we need to be able to do this for the components as potentially users can just use the styles without the behavior and manage the behavior on their own right?